src/payment/strategies/paypal-commerce/paypal-commerce-sdk.ts

```diff
@@ -59,13 +59,45 @@ export interface PaypalButtonStyleOptions {
 export interface ButtonsOptions {
     style?: PaypalButtonStyleOptions;
     fundingSource?: string;
-    createOrder?(): Promise<string>;
+    createOrder?(_: any, actions: any): Promise<string>;
```

Comment:
Is there any way to specify type instead of any?

Reply:
fixed